src/main/java/com/google/jenkins/plugins/k8sengine/KubeConfig.java

```diff
@@ -178,6 +178,7 @@ static String clusterServer(Cluster cluster) {
             new ImmutableMap.Builder<String, Object>()
                 .put("cluster", currentContext)
                 .put("user", currentContext)
+                .put("namespace", "default")
```

Comment:
Namespace is always default in the kube config? Will it ever need to be the namespace specified by the user?

Reply:
Actually, does putting namespace into --namespace into kubectl take care of this (where namespace is not default)?

Reply:
When the user provides a namespace, that gets passed as the `--namespace` flag. This is for the cases where user leaves the namespace blank and also doesn't provide it in the manifest in order to have defined behavior then. If the namespace is not set through either of these methods then the namespace used *should* be default but kubectl can sometimes pick up a namespace from a service account. This ensures that it will be default if not included in the flag or manifest.

Reply:
Cool, just wanted to ensure that even if the kube config's namespace is set to default that the --namespace "overrides" the default we put in the kube config.